### PR TITLE
WIP: Add exemplar support tasks

### PR DESCRIPTION
**Pull request by 0atman**
What
----

Moved some relevant lines over from https://docs.google.com/document/d/14ulAP1DbuK_NmNE2mZTVRZkg3wTw2deduZD74xGMjSc/edit?ts=5b7e7c6f#

Looks like this in the UI:

Screenshot from 2021-06-16 14-18-10

How to review
-------------

Look good? In the right place?

Who can review
--------------

Would love to hear from content people, as well as people who will use this runbook

### CODE REVIEW

File: source/guides/common_support_tasks.html.md [hunk 1]
```diff
@@ -43,3 +43,29 @@ Factors in their decision:
 - if the email account belonging to the absent org manager can be made active again to send an email authorising the change.
 
 We will also email every user currently in the organisation to explain what has been done.
+
+
+# Common Support questions
+
+## Which Databases do you support?
```
Comment (cmcnallygds) [inline source/guides/common_support_tasks.html.md:50]: I would have databases in lowercase here.

File: source/guides/common_support_tasks.html.md [hunk 2]
```diff
@@ -43,3 +43,29 @@ Factors in their decision:
...
+
+- Postgres
+- MySQL
+- Elasticsearch
+- Redis
+
+## Some plans in the marketplace are displaying as free, what does this mean?
```
Comment (cmcnallygds) [inline source/guides/common_support_tasks.html.md:57]:
```suggestion
## Some plans in the marketplace are displaying as free. What does this mean?
```

File: source/guides/common_support_tasks.html.md [hunk 3]
```diff
@@ -43,3 +43,29 @@ Factors in their decision:
...
+
+This is a horrible bit of CF UI: "free" really means "free for 3 months while you are in trial, paid after"
```
Comment (cmcnallygds) [inline source/guides/common_support_tasks.html.md:59]:
```suggestion
This is a horrible bit of Cloud Foundry UI: "free" really means "free for 3 months while you are in trial, and then you need to pay."
```

File: source/guides/common_support_tasks.html.md [hunk 4]
```diff
@@ -43,3 +43,29 @@ Factors in their decision:
...
+
+
+## Can I get direct access to the RDS database instances?
+
+No, the high level abstraction of the GOV.UK PaaS hides the details of the database inside the PaaS service.
```
Comment (cmcnallygds) [inline source/guides/common_support_tasks.html.md:64]:
```suggestion
No – the high level abstraction of the GOV.UK PaaS hides the details of the database inside the PaaS service. 
```

File: source/guides/common_support_tasks.html.md [hunk 5]
```diff
@@ -43,3 +43,29 @@ Factors in their decision:
...
+## Some plans in the marketplace are displaying as free, what does this mean?
+
+This is a horrible bit of CF UI: "free" really means "free for 3 months while you are in trial, paid after"
+
+
+## Can I get direct access to the RDS database instances?
+
+No, the high level abstraction of the GOV.UK PaaS hides the details of the database inside the PaaS service. 
+
+## What is your elasticsearch offering?
```
Comment (cmcnallygds) [inline source/guides/common_support_tasks.html.md:66]:
```suggestion
## What is your Elasticsearch offering?
```

File: source/guides/common_support_tasks.html.md [hunk 6]
```diff
@@ -43,3 +43,29 @@ Factors in their decision:
...
+We currently offer elasticsearch in private beta for v5 and v6
```
Comment (cmcnallygds) [inline source/guides/common_support_tasks.html.md:67]:
```suggestion
We currently offer Elasticsearch in private beta for v5 and v6
```

File: source/guides/common_support_tasks.html.md [hunk 7]
```diff
@@ -43,3 +43,29 @@ Factors in their decision:
...
+
+## When will your elasticsearch reach public beta?
```
Comment (cmcnallygds) [inline source/guides/common_support_tasks.html.md:69]:
```suggestion
## When will your Elasticsearch reach public beta?
```

File: source/guides/common_support_tasks.html.md [hunk 8]
```diff
@@ -43,3 +43,29 @@ Factors in their decision:
...
+We are currently in private beta and are currently testing the new service with tenants. We are confident that we have a reliable service and are working to solicit feedback from beta user before making the appliation more widely available.
```
Comment (cmcnallygds) [inline source/guides/common_support_tasks.html.md:70]:
```suggestion
We are currently in private beta and are currently testing the new service with tenants. We are confident that we have a reliable service and are working to solicit feedback from beta users before making the application more widely available.
```

File: source/guides/common_support_tasks.html.md [hunk 9]
```diff
@@ -43,3 +43,28 @@ Factors in their decision:
 - if the email account belonging to the absent org manager can be made active again to send an email authorising the change.
 
 We will also email every user currently in the organisation to explain what has been done.
+
+
+# Common Support questions
+
+## Which Databases do you support?
```
Comment (pauldougan) [inline source/guides/common_support_tasks.html.md:50]:
```suggestion
## Which databases do you support?
```

File: source/guides/common_support_tasks.html.md (hunk 8 above)
Comment (pauldougan) [reply source/guides/common_support_tasks.html.md:70]: PaaS is now a [live service](https://www.gov.uk/service-standard-reports/gov-dot-uk-platform-as-a-service-paas-live-assessment) so we should remove any mention of private beta

File: source/guides/common_support_tasks.html.md [hunk 10]
```diff
@@ -43,3 +43,28 @@ Factors in their decision:
...
+
+This is a horrible bit of Cloud Foundry UI: "free" really means "free for 3 months while you are in trial, and then you need to pay."
+
+
+## Can I get direct access to the RDS database instances?
+
+No – the high level abstraction of the GOV.UK PaaS hides the details of the database inside the PaaS service. 
+
+## What is your Elasticsearch offering?
+We currently offer Elasticsearch in private beta for v5 and v6
```
Comment (pauldougan) [inline source/guides/common_support_tasks.html.md:67]:
```suggestion
We currently offer Elasticsearch 7 
```

File: source/guides/common_support_tasks.html.md [hunk 11]
```diff
@@ -43,3 +43,28 @@ Factors in their decision:
...
+## Which databases do you support?
+
+- Postgres
+- MySQL
+- Elasticsearch
+- Redis
+
+## Some plans in the marketplace are displaying as free. What does this mean?
+
+This is a horrible bit of Cloud Foundry UI: "free" really means "free for 3 months while you are in trial, and then you need to pay."
```
Comment (pauldougan) [inline source/guides/common_support_tasks.html.md:59]:
```suggestion
This is a horrible bit of Cloud Foundry user interface: "free" really means "free for 3 months while you are in trial, and then you need to pay."
```

File: source/guides/common_support_tasks.html.md [hunk 12]
```diff
@@ -43,3 +43,28 @@ Factors in their decision:
 - if the email account belonging to the absent org manager can be made active again to send an email authorising the change.
 
 We will also email every user currently in the organisation to explain what has been done.
+
+
+# Common Support questions
+
+## Which databases do you support?
+
+- Postgres
```
Comment (pauldougan) [inline source/guides/common_support_tasks.html.md:52]:
```suggestion
- Postgres (RDS)
```

File: source/guides/common_support_tasks.html.md [hunk 13]
```diff
@@ -43,3 +43,28 @@ Factors in their decision:
...
+- MySQL
```
Comment (pauldougan) [inline source/guides/common_support_tasks.html.md:53]:
```suggestion
- MySQL (RDS)
```

File: source/guides/common_support_tasks.html.md [hunk 14]
```diff
@@ -43,3 +43,28 @@ Factors in their decision:
...
+
+
+## Can I get direct access to the RDS database instances?
+
+No – the high level abstraction of the GOV.UK PaaS hides the details of the database inside the PaaS service. 
+
+## What is your Elasticsearch offering?
+We currently offer Elasticsearch 7 
+
+## When will your Elasticsearch reach public beta?
```
Comment (pauldougan) [inline source/guides/common_support_tasks.html.md:69]:
```suggestion
## What is the status of the Elasticsearch backing service?
```

File: source/guides/common_support_tasks.html.md [hunk 15]
```diff
@@ -43,3 +43,28 @@ Factors in their decision:
...
+We are currently in private beta and are currently testing the new service with tenants. We are confident that we have a reliable service and are working to solicit feedback from beta user before making the appliation more widely available.
```
Comment (pauldougan) [inline source/guides/common_support_tasks.html.md:70]:
```suggestion
GOV.UK PaaS is [live service](https://www.gov.uk/service-standard-reports/gov-dot-uk-platform-as-a-service-paas-live-assessment)
```

File: source/guides/common_support_tasks.html.md [hunk 16]
```diff
@@ -43,3 +43,28 @@ Factors in their decision:
...
+
+This is a horrible bit of Cloud Foundry user interface: "free" really means "free for 3 months while you are in trial, and then you need to pay."
+
+
+## Can I get direct access to the RDS database instances?
+
+No – the high level abstraction of the GOV.UK PaaS hides the details of the database inside the PaaS service. 
+
+## What is your Elasticsearch offering?
+We currently offer Elasticsearch 7
```
Comment (pauldougan) [inline source/guides/common_support_tasks.html.md:67]:
```suggestion
We currently offer [Elasticsearch 7](https://admin.cloud.service.gov.uk/marketplace/b98f53e7-85a7-4964-bace-9ce27fac142a)
```

File: source/guides/common_support_tasks.html.md [hunk 17]
```diff
@@ -43,3 +43,28 @@ Factors in their decision:
 - if the email account belonging to the absent org manager can be made active again to send an email authorising the change.
 
 We will also email every user currently in the organisation to explain what has been done.
+
+
+# Common Support questions
+
+## Which databases do you support?
+
+- Postgres (RDS)
```
Comment (pauldougan) [inline source/guides/common_support_tasks.html.md:52]:
```suggestion
- [Postgres](https://admin.cloud.service.gov.uk/marketplace/efadb775-58c4-4e17-8087-6d0f4febc489) (RDS)
```

File: source/guides/common_support_tasks.html.md [hunk 18]
```diff
@@ -43,3 +43,28 @@ Factors in their decision:
...
+- MySQL (RDS)
```
Comment (pauldougan) [inline source/guides/common_support_tasks.html.md:53]:
```suggestion
- [MySQL](https://admin.cloud.service.gov.uk/marketplace/8ffb13cb-705f-4cee-9ffd-5a950f9f3048) (RDS)
```

File: source/guides/common_support_tasks.html.md [hunk 19]
```diff
@@ -43,3 +43,28 @@ Factors in their decision:
...
 We will also email every user currently in the organisation to explain what has been done.
+
+
+# Common Support questions
+
+## Which databases do you support?
+
+- [Postgres](https://admin.cloud.service.gov.uk/marketplace/efadb775-58c4-4e17-8087-6d0f4febc489) (RDS)
+- [MySQL](https://admin.cloud.service.gov.uk/marketplace/8ffb13cb-705f-4cee-9ffd-5a950f9f3048) (RDS)
+- Elasticsearch
```
Comment (pauldougan) [inline source/guides/common_support_tasks.html.md:54]:
```suggestion
- [Elasticsearch](https://admin.cloud.service.gov.uk/marketplace/b98f53e7-85a7-4964-bace-9ce27fac142a)
```

File: source/guides/common_support_tasks.html.md [hunk 20]
```diff
@@ -43,3 +43,28 @@ Factors in their decision:
...
+
+
+# Common Support questions
+
+## Which databases do you support?
+
+- [Postgres](https://admin.cloud.service.gov.uk/marketplace/efadb775-58c4-4e17-8087-6d0f4febc489) (RDS)
+- [MySQL](https://admin.cloud.service.gov.uk/marketplace/8ffb13cb-705f-4cee-9ffd-5a950f9f3048) (RDS)
+- [Elasticsearch](https://admin.cloud.service.gov.uk/marketplace/b98f53e7-85a7-4964-bace-9ce27fac142a)
+- Redis
```
Comment (pauldougan) [inline source/guides/common_support_tasks.html.md:55]:
```suggestion
- [Redis](https://admin.cloud.service.gov.uk/marketplace/0e4b6fb6-243c-4024-8604-41635d1233cc)
```

File: source/guides/common_support_tasks.html.md [hunk 21]
```diff
@@ -43,3 +43,28 @@ Factors in their decision:
...
+
+# Common Support questions
+
+## Which databases do you support?
+
+- [Postgres](https://admin.cloud.service.gov.uk/marketplace/efadb775-58c4-4e17-8087-6d0f4febc489) (RDS)
+- [MySQL](https://admin.cloud.service.gov.uk/marketplace/8ffb13cb-705f-4cee-9ffd-5a950f9f3048) (RDS)
+- [Elasticsearch](https://admin.cloud.service.gov.uk/marketplace/b98f53e7-85a7-4964-bace-9ce27fac142a)
+- [Redis](https://admin.cloud.service.gov.uk/marketplace/0e4b6fb6-243c-4024-8604-41635d1233cc)
+
```
Comment (pauldougan) [inline source/guides/common_support_tasks.html.md:56]:
```suggestion
- [Influxdb](https://admin.cloud.service.gov.uk/marketplace/0b7da7a9-2fd7-4dca-9e25-0cf8eddd0f88)

```

File: source/guides/common_support_tasks.html.md [hunk 22]
```diff
@@ -43,3 +43,29 @@ Factors in their decision:
...
+## Can I get direct access to the RDS database instances?
+
+No – the high level abstraction of the GOV.UK PaaS hides the details of the database inside the PaaS service. 
+
+## What is your Elasticsearch offering?
+We currently offer [Elasticsearch 7](https://admin.cloud.service.gov.uk/marketplace/b98f53e7-85a7-4964-bace-9ce27fac142a)
+
+## What is the status of the Elasticsearch backing service?
+GOV.UK PaaS is [live service](https://www.gov.uk/service-standard-reports/gov-dot-uk-platform-as-a-service-paas-live-assessment)
```
Comment (pauldougan) [inline source/guides/common_support_tasks.html.md:71]:
```suggestion
GOV.UK PaaS is a [live service](https://www.gov.uk/service-standard-reports/gov-dot-uk-platform-as-a-service-paas-live-assessment)
```

**Pull request merged**